corehq/pillows/mappings/domain_mapping.py

```diff
@@ -192,6 +192,8 @@
                 'is_snapshot': {'type': 'boolean'},
                 'is_test': {'type': 'string'},
                 'is_starter_app': {'type': 'boolean'},
+                'last_modified': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
+                                  'type': 'date'},
```

Comment:
Ran

```python
DOMAIN_MAPPING['properties']['date_created'] == DOMAIN_MAPPING['properties']['last_modified']
```

in manage shell and checked that [there really is a property on `Domain` called `last_modified](https://github.com/dimagi/commcare-hq/blob/ee7bed5c13fadabb5dd6641db555698997efbe59/corehq/apps/domain/models.py#L340)